src/styles/blocks/button.scss

```diff
@@ -0,0 +1,27 @@
+.button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
```

Comment:
Don't hardcode width, let's container with paddings define how much space for this button needed

- Don't fix container size (if there is no such a requirement)
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/8ec9d8fc-d8ee-4f93-8910-822111d54e05)

Reply:
Not fixed